Header header
int32 waypoint_num

---

Header header
string status